runners/spark/src/main/java/org/apache/beam/runners/spark/SparkCommonPipelineOptions.java

```diff
@@ -32,6 +32,8 @@
 public interface SparkCommonPipelineOptions
     extends PipelineOptions, StreamingOptions, ApplicationNameOptions {
   String DEFAULT_MASTER_URL = "local[4]";
+  String DEFAULT_SPARK_HISTORY_DIR = "/tmp/spark-events/";
```

Comment:
Remove `DEFAULT_SPARK_HISTORY_DIR` and `DEFAULT_EVENT_LOG_ENABLED` since they are unused now.

Reply:
Ok, I removed it.